packages/portal/backend/src/custom/CustomCourseController.ts

```diff
@@ -0,0 +1,917 @@
+import * as crypto from 'crypto';
```

Comment:
Wouldn't it make more sense for the SDMM Controller and Route logic to be put in a plugin? They are already defined as CustomCourseController and CustomRouteController respectively. Was it intentional to include them in this PR?